### PR TITLE
Fix bug: chain id is hex string, so it should has 0x-prefix.

### PR DESCRIPTION
**Pull request by chaoticlonghair**

### CODE REVIEW

**Review by kaikai1024 — APPROVED** [review]

File: cita-types/src/traits.rs [hunk 1]
```diff
@@ -93,7 +93,8 @@ macro_rules! add_cita_funcs {
 
             #[inline]
             fn lower_hex_with_0x(&self) -> String {
-                format!("{:#x}", self)
+                // TODO: https://github.com/paritytech/primitives/pull/37
```
Comment (u2) [inline cita-types/src/traits.rs:96]: This PR has been merged, so this comment can be removed?

**Review by keroro520 — APPROVED** [review]
Nice !

File: cita-types/src/traits.rs (hunk 1 above)
Comment (chaoticlonghair) [reply cita-types/src/traits.rs:96]: When I added that TODO few month ago, that PR was merged, But they did not release it.
Now, there are some troubles for upgrading that crate.
(I checked, the version we used is not fixed)

Comment (u2) [reply cita-types/src/traits.rs:96]: ok

**Review by u2 — APPROVED** [review]

**Pull request merged**